### ISSUE TITLE
--noUnusedLocals gives false negatives for unused constructor parameters

### ISSUE THREAD

**Issue by aleksey-bykov** [Microsoft/TypeScript#9602]
nightly build

```typescript
class A {
    constructor(
        used: string // <-- no problem, expected an error
    ) {
    }
}

new A('');
```

**Comment by aleksey-bykov** [comment]
--noUnusedParameters works
